practice-two/src/style/sections/project.css

```diff
@@ -0,0 +1,168 @@
+.project {
+  padding-top: 29px;
+  padding-bottom: 50px;
+}
+
+.project__contents {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 35px;
+  margin-top: 20px;
+}
+
+.project__item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.project__item-description {
+  display: flex;
+  flex-direction: column;
+  margin-top: 16px;
+}
+
+.project__item-title {
+  text-align: center;
+  font-size: var(--font-size-xs);
+  font-style: var(--font-style-default);
+  font-weight: var(--font-weight-primary);
+  line-height: 24px;
+  text-transform: uppercase;
+}
+
+.project__item-label {
+  display: block;
+}
+
+.project__item-author {
+  text-align: center;
+}
+
+.btn--project {
+  margin-top: 10px;
```

Comment:
Move this style into project__item-description class, use margin-bottom instead of